istio-control/istio-config/templates/service.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   ports:
   - port: 443
     name: https-validation
-  - port: 15014
+  - port:  {{ .Values.global.monitoringPort }}
```

Comment:
Same. If we can hardcode 9901 and 443, there is no reason to customize monitoringPort.